src/Microsoft.VisualStudio.Editors/PropPages/AdvBuildSettingsPropPage.vb

```diff
@@ -173,7 +173,9 @@ Namespace Microsoft.VisualStudio.Editors.PropertyPages
 
                 Dim iBaseAddress As UInteger
 
-                If (TypeOf (value) Is UInteger) Then
+                Dim throwAwayObject As Decimal = Nothing
+                If (TypeOf (value) Is UInteger OrElse
+                    (CpsPropertyDescriptorWrapper.IsAnyCpsComponent(m_Objects) AndAlso Decimal.TryParse(DirectCast(value, String), throwAwayObject))) Then
                     iBaseAddress = CUInt(value)
```

Comment:
It is the value from DynamicType since we get that from `DynamicTypeBrowseObject`